openhub/tests/commands/test_import_affiliated_committers_data.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management import call_command
+from django.test import TestCase
+
+from openhub.models import AffiliatedCommitter
+
+
+class ImportAffiliatedCommitterDataTest(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        call_command('import_affiliated_committers_data')
+
+    def test_command_import_affiliated_committers_data(self):
+        a_committers = AffiliatedCommitter.objects.all()
```

Comment:
```python
committer = AffiliatedCommitter.objects.get(name='John Vandenberg')
```
This looks better to me, and is consistent with what you do in `test_import_organization_data.py`